main.py

```diff
@@ -86,12 +132,30 @@ async def cleardb(ctx):
 
 @bot.command()
 @commands.check(has_required_role)
-async def endairdrop(ctx, num_winners: int):
+async def endairdrop(ctx, num_winners: int, cms_amount: int):
+    if num_winners <= 0:
+        await ctx.send("Please provide a valid number of winners.")
+        return
+
     db = load_db()
     winners = random.sample(db, min(num_winners, len(db)))
     winners_list = "\n".join([f"{index + 1}) <@{winner['user_id']}> - {winner['wallet']}" for index, winner in enumerate(winners)])
-    announcement = f"Congrats! You have won the Cmusic airdrop! Your funds should be automatically deposited into the wallets you provided. Here are the winners:\n{winners_list} \n\n New airdrop has started, re-enter your wallets!"
+    
+    results = []
+    transaction_explorer_links = []
+    for winner in winners:
+        sendresult = send_coins(rpc_user, rpc_password, rpc_port, rpc_ip, from_address, winner['wallet'], cms_amount)
+        result = sendresult.get('result')
+        results.append(result)
+        transaction_explorer_links.append(f"[Transaction Explorer](https://explorer.cmusic.ai/tx/{result})")
```

Comment:
Can we make the explorer link an environmental variable.

Reply:
done